### ISSUE TITLE
application/octect-stream response is not handled 

### ISSUE THREAD

**Issue by appunni-dishq** [wkeeling/selenium-wire#20]
```
DEBUG http://seleniumwire/requests 200
DEBUG:seleniumwire.proxy.handler:http://seleniumwire/requests 200
DEBUG http://seleniumwire/response_body?request_id=6f7ba003-9c3c-4b58-a837-73c853488590 200
DEBUG:seleniumwire.proxy.handler:http://seleniumwire/response_body?request_id=6f7ba003-9c3c-4b58-a837-73c853488590 200
----------------------------------------
Exception happened during processing of request from ('127.0.0.1', 36218)
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/home/user/.virtualenvs/project/lib/python3.5/site-packages/seleniumwire/proxy/proxy2.py", line 65, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
    self.handle()
  File "/usr/lib/python3.5/http/server.py", line 422, in handle
    self.handle_one_request()
  File "/usr/lib/python3.5/http/server.py", line 410, in handle_one_request
    method()
  File "/home/user/.virtualenvs/project/lib/python3.5/site-packages/seleniumwire/proxy/handler.py", line 125, in do_GET
    super().do_GET()
  File "/home/user/.virtualenvs/project/lib/python3.5/site-packages/seleniumwire/proxy/proxy2.py", line 147, in do_GET
    self.admin_handler()
  File "/home/user/.virtualenvs/project/lib/python3.5/site-packages/seleniumwire/proxy/handler.py", line 36, in admin_handler
    self._get_response_body(**params)
  File "/home/user/.virtualenvs/project/lib/python3.5/site-packages/seleniumwire/proxy/handler.py", line 72, in _get_response_body
    self._send_body(body)
  File "/home/user/.virtualenvs/project/lib/python3.5/site-packages/seleniumwire/proxy/handler.py", line 77, in _send_body
    self._send_response(body, 'application/octet-stream')
  File "/home/user/.virtualenvs/project/lib/python3.5/site-packages/seleniumwire/proxy/handler.py", line 115, in _send_response
    self.wfile.write(body)
  File "/usr/lib/python3.5/socket.py", line 593, in write
    return self._sock.send(b)
TypeError: a bytes-like object is required, not 'str'
```

**Comment by wkeeling** [comment]
Thanks @appunni-dishq . This is strange as I was under the impression that the response body should be of type `bytes` when it is read, not `str`, although it is possible it was converted somewhere. 

Out of curiosity, when you asked for the response body, what kind of object was that response for - a binary object such as an image, or a text based object such as text file?

In the meantime, I'll fix the code to handle the possibility of a `str` when retrieving the response body.

**Comment by wkeeling** [comment]
@appunni-dishq - this has been fixed in master. Are you able to pull the latest master and it confirm it resolves this issue?

**Comment by appunni-dishq** [comment]
@wkeeling I will let you know by tomorrow

**Comment by wkeeling** [comment]
Fixed in 1e9401ed615ec83962e7739ae697b704a8a74067
